tox.ini

```diff
@@ -9,7 +9,7 @@ tox_pip_extensions_ext_pip_custom_platform = true
 tox_pip_extensions_ext_venv_update = true
 
 [testenv]
-basepython = /usr/bin/python2.7
+basepython = python3.6
```

Comment:
the full path here was important (so you can recreate the virtualenv while it is activated -- a relatively common workflow)